### PR TITLE
Adds sidebar as a documentation item

### PR DESCRIPTION
**Pull request by isralduke**
This PR:

- Adds the sidebar as a documentation item
- Edits relevant pages and information architecture as necessary

### CODE REVIEW

File: docs/components/app-screen.md [hunk 1]
```diff
@@ -35,54 +35,15 @@ Everything considered part of the App Screen is placed into the `id=page-wrapper
 
 ## Sidebar with Navigation
 
-The first child element is the Sidebar with Navigation. This is the `nav`. For small devices like smartphones, it makes responsive adjustments. Add Top Level and Child Level items as needed.
+The first child element of the `id=page-wrapper` is the [Sidebar with Navigation](/components/sidebar-nav/). This is the `nav`. For small devices like smartphones, it makes responsive adjustments. Add Top Level and Child Level items as needed.
```
Comment (frogpad7) [inline docs/components/app-screen.md:38]: It would be nice if this link reference was replaced everywhere on the page (like in the Topbar section text).

Comment (frogpad7) [reply docs/components/app-screen.md:38]: [Sidebar with Navigation](/components/sidebar-nav/)

File: docs/components/sidebar-nav.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  img: /img/illustrations/illus-share-social.svg
+---
+  
+## Best Practice
+
+The first child element of the `id=page-wrapper` is the Sidebar with Navigation. This is the `nav`. For small devices like smartphones, it makes responsive adjustments. Add Top Level and Child Level items as needed.
+
+- Used in the [App Screen](/components/app-screen/).
+- Presents only two levels of navigation.
+- Current pages should get the class `active` added to the class list of the `li` and `a` which hold the link.
```
Comment (frogpad7) [inline docs/components/sidebar-nav.md:20]: This only applies to the child pages at the present moment (no indicator changes on `sidebar-item`); might want to also mention that the `sidebar-submenu` does not get revealed unless its display goes to `block`.

File: docs/components/sidebar-nav.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: Sidebar with Navigation
```
Comment (frogpad7) [inline docs/components/sidebar-nav.md:2]: Not sure if the url (which is the name of the file) should be spelled out like in the title of the page.

File: docs/_includes/markup/sidebar-nav.njk [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+                  <span>Child Nav Item</span>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </li>
+      </ul>
+    </div>
+  </div>
+  <div class="sidebar-footer">
```
Comment (frogpad7) [inline docs/_includes/markup/sidebar-nav.njk:41]: Should this be noted on any of the pages for its usage?

File: docs/components/app-screen.md [hunk 5]
```diff
@@ -35,54 +35,15 @@ Everything considered part of the App Screen is placed into the `id=page-wrapper
...
 
-- Current pages get the class `active` added to the class list of the `li` and `a` which hold the link.
+- Used in the App Screen.
+- Presents only two levels of navigation.
+- Current pages should get the class `active` added to the class list of the `li` and `a` which hold the link.
```
Comment (frogpad7) [inline docs/components/app-screen.md:42]: This only applies to the child pages at the present moment (no indicator changes on `sidebar-item`); might want to also mention that the `sidebar-submenu` does not get revealed unless its display goes to `block`.

**Review by frogpad7 — CHANGES_REQUESTED** [review]
Just a few things to help make the presentation of the sidebar navigation more clear.

File: docs/components/app-screen.md (hunk 1 above)
Comment (isralduke) [reply docs/components/app-screen.md:38]: not worrying about that

File: docs/components/app-screen.md (hunk 5 above)
Comment (isralduke) [reply docs/components/app-screen.md:42]: that’s in a bullet item to come

File: docs/_includes/markup/sidebar-nav.njk (hunk 4 above)
Comment (isralduke) [reply docs/_includes/markup/sidebar-nav.njk:41]: noted

File: docs/components/sidebar-nav.md (hunk 2 above)
Comment (isralduke) [reply docs/components/sidebar-nav.md]: noted in a bullet item to come

**Review by frogpad7 — DISMISSED** [review]
Great text info updates!

**Review by rltucker — COMMENTED** [review]
App Screen Section - line 20 spelling "beig"

**Review by rltucker — APPROVED** [review]
Approved

**Pull request merged**